README.md

```diff
@@ -0,0 +1,26 @@
+# Jenkins Community Kubernetes Helm Charts
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+This functionality is in beta and is subject to change. The code is provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.
+Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+Once Helm is set up properly, add the repo as follows:
+
+```console
+helm repo add jenkinsci https://jenkinsci.github.io/helm-charts
```

Comment:
we can probably setup a CNAME if that would be preferred?

https://charts.jenkins.io ?

e.g. send a PR here:
https://github.com/jenkins-infra/azure/pull/154/files

Reply:
Do you have a name in mind? I've been thinking of `charts.jenkins.io`. Using that the URL would be `helm repo add jenkinsci https://charts.jenkins.io/helm-charts`.

Reply:
that was the name i had in mine 😄

Reply:
Created a PR for it: https://github.com/jenkins-infra/azure/pull/156

Reply:
It's been changed to helm repo add jenkinsci https://charts.jenkins.io